Tools/Scripts/hooks/prepare-commit-msg

```diff
@@ -10,6 +10,29 @@ SCRIPTS = os.path.dirname(os.path.dirname(LOCATION))
 
 sys.path.append(SCRIPTS)
 from webkitpy.common.checkout.diff_parser import DiffParser
+from webkitbugspy import radar
+
+
+def get_bugs_string():
+    """Determines what bug URLs to fill or suggest in a WebKit commit message."""
+    env = os.environ
+    need_the_bug_url = 'Need the bug URL (OOPS!).'
+    need_the_radar = 'Heavily recommended: include a Radar link with no title (OOPS!).'
```

Comment:
Would just change the to `Include a Radar link (OOPS!)`

Reply:
This would be a bit awkward to deal with until automatic radar importing is enabled w.r.t bugzilla right?

Reply:
We already have some automatic radar importing, in that if you give `git-webkit` a bugzilla which is cloned into radar, we'll pick up the radar too. @alancoon and I do need to make this particular behavior cover a few more cases.